foam_controlwrapper/tests/test_foam_files.py

```diff
@@ -0,0 +1,100 @@
+""" test_foam_files module
+
+This module contains the unitary tests for the
+foam_files module functionalities
+
+"""
+
+import unittest
+
+from simphony.core.cuba import CUBA
+from simphony.core.data_container import DataContainer
+from foam_controlwrapper.foam_files import FoamFiles
+from foam_controlwrapper.foam_controlwrapper import FoamControlWrapper
+from simphony.io.h5_cuds import H5CUDS
+import os
+import shutil
+
+
+class FoamFilesTestCase(unittest.TestCase):
```

Comment:
The included test methods are not verify the expected behaviour of the code (please see comments in https://github.com/simphony/simphony-openfoam/pull/10/files#r27658560)